ui/src/components/footer.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { useAdminServiceGetApiV1AdminContentManagementSectionsBySectionId } from '@/api/queries';
+import { MarkdownRenderer } from '@/components/markdown-renderer.tsx';
+
+function extractCompanyTag(markdown: string): string | undefined {
+  const match = markdown.match(/:::\s?company\s+(.+?)\s+:::/s);
+  return match ? match[1] : undefined;
+}
+
+function extractColumns(markdown: string): string[] {
+  const regex = /:::\s?column\s+([\s\S]*?)\s+:::/g;
```

Comment:
Why is this regex actually different from the "company" one? Don't both "company" and "column" markers follow exactly the same syntax?